### PR TITLE
Use wasm pack build

### PR DESCRIPTION
**Pull request by d-dorazio**
should solve https://github.com/wasm-tool/wasm-pack-plugin/issues/27

The diff is a bit noisy because I took the freedom to update some dependencies and to format the code a bit but the relevant commit is the last one. I can revert the other commits if they're out of scope.

### CODE REVIEW

File: example/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "clean": "rimraf dist pkg target"
   },
   "devDependencies": {
-    "@wasm-tool/wasm-pack-plugin": "^0.0.5",
+    "@wasm-tool/wasm-pack-plugin": "../",
```
Comment (xtuc) [inline example/package.json]: I don't think we want this, it means that you can not run the example out of this repo anymore.

File: plugin.js [hunk 2]
```diff
@@ -132,7 +145,6 @@ function runProcess(bin, args, options) {
   });
 }
 
-module.exports = function() {
-};
+module.exports = function () {};
```
Comment (xtuc) [inline plugin.js]: Why do we even have that? It's overriden two lines under.

**Comment by d-dorazio** [comment]
@xtuc I plan to open another small PR soonish to configure whether `wasm-pack` should emit typescript bindings or not(as far as I can tell [they're not generated now](https://github.com/wasm-tool/wasm-pack-plugin/blob/master/plugin.js#L110)) so I guess it's simpler to release after that.

**Comment by xtuc** [comment]
Ok cool, we can wait I guess.

**Pull request merged**